judge/views/organization.py

```diff
@@ -47,7 +48,7 @@ def dispatch(self, request, *args, **kwargs):
         try:
             return super(OrganizationMixin, self).dispatch(request, *args, **kwargs)
         except Http404:
-            key = kwargs.get(self.slug_url_kwarg, None)
+            key = kwargs.get(self.slug_url_kwarg, None) if hasattr(self, 'slug_url_kwarg') else None
```

Comment:
I don't know if we want this `hasattr` check. For `AttributeError` to occur, the site would be beyond broken.

Reply:
Fixed by #2240.